.dev/Project.toml

```
@@ -1,2 +1,5 @@
 [deps]
-JuliaFormatter = "98e50ef6-434e-11e9-1051-2b60c6c9e899"
\ No newline at end of file
+JuliaFormatter = "98e50ef6-434e-11e9-1051-2b60c6c9e899"
+
+[compat]
+JuliaFormatter = "=1"
```

```suggestion
JuliaFormatter = "1"
```

This includes any version in the 1.X series, but not in the 2.X (or 3.X)